src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableEntityClient.cs

```diff
@@ -59,7 +59,17 @@ public interface IDurableEntityClient
         /// </summary>
         /// <param name="query">Return entity instances that match the specified query conditions.</param>
         /// <param name="cancellationToken">Cancellation token that can be used to cancel the query operation.</param>
+        /// <param name="connectionDetails">The storage account connection details.</param>
         /// <returns>Returns a page of entity instances and a continuation token for fetching the next page.</returns>
+        Task<EntityQueryResult> ListEntitiesAsync(EntityQuery query, CancellationToken cancellationToken, DurableConnectionDetails connectionDetails = null);
```

Comment:
I avoided/removed my changes to IDurableEntityClient that added new methods with DurableConnectionDetails as an optional parameter for the other methods in this interface to avoid errors that a user's method call was ambiguous. Adding these new methods would be a breaking change that we can put off for now.